### PR TITLE
Polished counter Example - querying the Network ( NCTL and Testnet )

### LINKED ISSUE #97: Tutorials/Beginner tutorials/Query the Casper Network
.

### PR DESCRIPTION
**Pull request by jonas089**
Polished "querying the Network" in counter example ( NCTL and Testnet ). Added an example for the default configuration ( http://localhost:11101 and a link to peers on testnet.cspr.live to make the --node-address more clear.

### CODE REVIEW

**Comment by ghost** [comment]
Rebased to new structure.

File: source/docs/casper/resources/tutorials/beginner/counter-testnet/commands.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 
 ```rust
 casper-client get-state-root-hash --node-address [NODE_SERVER_ADDRESS]
 ```
 
+Example:
+```rust
+casper-client get-state-root-hash --node-address http://[IP]:7777
+```
+You can find a list of Testnet IP addresses [here](https://testnet.cspr.live/tools/peers)
```
Comment (ghost) [inline source/docs/casper/resources/tutorials/beginner/counter-testnet/commands.md]:
```suggestion
You can find a list of Testnet IP addresses at [CSPR live](https://testnet.cspr.live/tools/peers).
```

File: source/docs/casper/resources/tutorials/beginner/counter/commands.md [hunk 2]
```diff
@@ -34,7 +34,12 @@ casper-client query-global-state \
 
 This command allows you to query the state of a Casper network at a given moment in time, which is specified by the _state-root-hash_ described above.
 
--   The _node-address_ is the server (localhost when running on a local NCTL network).
+-   The _node-address_ is the server's _IP:PORT_, where port is usually the default RPC-port when setting up an NCTL network is _11101_ (IP == localhost when running on a local NCTL network).
+
+Example:
+```rust
+  casper-client ... --node-address http://localhost:11101
+```
```
Comment (ghost) [inline source/docs/casper/resources/tutorials/beginner/counter/commands.md]: I like that you introduced example URL for this tutorial :+1: However, I would make it inline to keep bullet list concise:
```suggestion
-   The _node-address_ is the location of the RPC endpoint, which is typically represented in the form `http://IP:PORT`. In this particular tutorial, for a default-configured NCTL network running locally, the address you can use is `http://localhost:11101`.

**Review by ghost — CHANGES_REQUESTED** [review]
I added 2 small suggestions.

**Comment by jonas089** [comment]
@andrzej-casper I committed your requested changes :) Thanks 👍🏻

**Review by ghost — APPROVED** [review]

**Pull request merged**